core/templates/components/state-editor/state-editor-properties-services/state-hints.service.spec.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit test for the State Hints service.
+ */
+
+import { TestBed } from '@angular/core/testing';
+import { UtilsService } from 'services/utils.service';
+import { AlertsService } from 'services/alerts.service';
+/* eslint-disable max-len */
+import { StateHintsService } from 'components/state-editor/state-editor-properties-services/state-hints.service';
+/* eslint-enable max-len */
+
+
+describe('State hints service', () => {
+  let shs: StateHintsService = null;
+  let alertsService : AlertsService;
+  let utilsService : UtilsService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [StateHintsService]
+    });
+
+    shs = TestBed.get(StateHintsService);
+  });
+
+
+  it('tests that the constructor should have been called', () =>{
```

Comment:
please start `it` block with `should` word.
```suggestion
  it('should constructor have been called', () =>{
```

Reply:
done